_posts/pccisme/2020-08-31-pccisme-First-Reflection.md

```diff
@@ -0,0 +1,36 @@
+## First Reflection
```

Comment:
Ah, I spoke too soon. This file needs a YAML header (the part with the `---` on top and bottom) for it to show up as a page in our website.

Reply:
No problem! And I just add the YAML header!